src/sas/qtgui/Perspectives/Fitting/FittingUtilities.py

```diff
@@ -550,7 +550,7 @@ def residualsData1D(reference_data, current_data, weights):
     residuals.ytransform = 'y'
     if reference_data.isSesans:
         residuals.xtransform = 'x'
-        residuals.xaxis('\\rm{z} ', 'A')
+        residuals.xaxis('\\delta ', 'A')
```

Comment:
Shouldn't residuals properties inherit from reference data, at least for the xaxis and xtransform?

Reply:
probably, but clearly beyond the simple scope of this PR. I guess that is part of the monster combo of plot refactoring and remaining work in sasdata project?